src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/transformation/model/request/DocumentType.java

```diff
@@ -1,8 +1,22 @@
 package uk.gov.hmcts.reform.bulkscan.orchestrator.client.transformation.model.request;
 
+import com.fasterxml.jackson.annotation.JsonValue;
+
 public enum DocumentType {
-    CHERISHED,
-    COVERSHEET,
-    FORM,
-    OTHER
+    CHERISHED("cherished"),
+    COVERSHEET("coversheet"),
+    FORM("form"),
+    OTHER("other");
+
+    private final String value;
+
+    DocumentType(String value) {
+        this.value = value;
+    }
+
+    @Override
+    @JsonValue
+    public String toString() {
+        return value;
+    }
```

Comment:
We need to check that it work in a test.

Reply:
it is checking as a regex providing only lower cases